scripts/website.js

```diff
@@ -79,22 +79,24 @@ const tests = [
 
 function deleteAllHtmlFiles() {
   try {
-    fs.unlinkSync('./index.html');
+    console.log('Delete', path.join(versionObj.versionedPath, 'index.html'));
+    fs.unlinkSync(path.join(versionObj.versionedPath, 'index.html'));
   } catch (err) {
     if (err.code !== 'ENOENT') {
       throw err;
     }
   }
   const foldersToClean = [
-    './docs',
-    './docs/tutorials',
-    './docs/typescript',
-    './docs/api',
-    './docs/source/_docs',
+    path.join('.', versionObj.versionedPath, 'docs'),
+    path.join('.', versionObj.versionedPath, 'docs', 'tutorials'),
+    path.join('.', versionObj.versionedPath, 'docs', 'typescript'),
+    path.join('.', versionObj.versionedPath, 'docs', 'api'),
+    path.join('.', versionObj.versionedPath, 'docs', 'source', '_docs'),
```

Comment:
as a suggestion, you should probably not use `'.'` and use [variable `cwd` instead](https://github.com/Automattic/mongoose/blob/140a118dda67cbf9d9ce62ba0eac1ad0efd13154/scripts/website.js#L16)